mnist/README.md

```diff
@@ -133,15 +133,18 @@ $ argo list
 NAME   STATUS   AGE   DURATION
 ```
 
-### Creating secrets for our workflow
+### Creating secrets for our workflow and setting AWS variables.
 For fetching and uploading data, our workflow requires S3 credentials. These credentials will be provided as kubernetes secrets:
 
 ```
 export S3_ENDPOINT=s3.us-west-2.amazonaws.com
```

Comment:
Maybe add a comment for minio here, like you did for the next line `#replace with your minio service endpoint ip for default minio installs`

Reply:
Added a comment and gave a minio example.